spec/cb/clients/expired_job_spec.rb

```diff
@@ -0,0 +1,46 @@
+# Copyright 2016 CareerBuilder, LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+require 'spec_helper'
+
+module Cb
+  describe Cb::Clients::ExpiredJob do
+    let(:headers) do
+      {
+        'Accept'=>'application/json', 
+        'Accept-Encoding'=>'deflate, gzip', 
+        'Authorization'=>'Bearer', 
+        'Content-Type'=>'application/json', 
+        'Developerkey'=> Cb.configuration.dev_key
+      }
+    end
```

Comment:
I am going to re-open the larger PR tomorrow.  I didn't want to cause a conflict on this one since I knew it was coming.